cmd/cstor-pool-mgmt/pool/pool.go

```diff
@@ -90,6 +97,29 @@ func CreatePool(cStorPool *apis.CStorPool) error {
 	return nil
 }
 
+func validateDiskCount(pool *apis.CStorPool) error {
+	poolType := pool.Spec.PoolSpec.PoolType
+	diskCount := len(pool.Spec.Disks.DiskList)
+	diskCountEnv := getDiskCountEnv()
+	if diskCountEnv == 0 {
+		diskCountEnv = defaultDiskCount
+	}
+
+	if diskCount < defaultDiskCount {
+		return fmt.Errorf("Expected minimum %v no of disks, but got %v for pool type: %v", defaultDiskCount, diskCount, poolType)
```

Comment:
We have moved to use errors package for this kind of stuff.
For example, 
https://github.com/openebs/maya/blob/master/cmd/maya-apiserver/spc-watcher/select_disk.go#L57
There are other examples in that file too ( using `errors.wrapf` etc)

Reply:
ok